README.md

```diff
@@ -1 +1,43 @@
-# Kaidan
+# Kaidan - Cross platform XMPP client
+
+Installation
+------------
+
+On Linux do the following:
+
+Create a working directory
+
+ * mkdir src
+ * cd src
+
+Fetch swift source
+
+ * wget https://swift.im/downloads/releases/swift-3.0/swift-3.0.tar.gz
+ * tar -xzvf swift-3.0.tar.gz
+ * cd swift-3.0/
+
+Install all dependencies to build swiften
+
+ * ./BuildTools/InstallSwiftDependencies.sh
+ * ./scons Swiften -j<Number of threads>
+
+Install dependencies to build Kaidan (example for Debian)
+
+ * sudo apt-get install zlib1g-dev libssl-dev libxml2-dev libstdc++-4.9-dev libqt5quick5 libqt5quickparticles5 libqt5quickwidgets5 libqt5qml5 libqt5network5 libqt5gui5 libqt5core5a qt5-default libglib2.0-dev libpthread-stubs0-dev
```

Comment:
`libstdc++-4.9-dev` is not existing in debian sid anymore. (but `libstdc++-5-dev` and `libstdc++-6-dev`)